CHANGELOG.md

```diff
@@ -64,6 +65,8 @@
 - Fixed a bug in `DataFrame.lineage.trace` to split the quoted feature view's name and version correctly.
 - Fixed a bug in `Column.isin` that caused invalid sql generation when passed an empty list.
 - Fixed a bug that fails to raise NotImplementedError while setting cell with list like item.
+- Fixed a bug in `session.read.csv` that caused an error when setting `PARSE_HEADER = True` in an externally defined file format.
+- Fixed a bug in `pyspark.sql.functions.array_remove` to remove is available in SQL but there is no corresponding API.
```

Comment:
move this up to 1.22.0 release section

Reply:
Again, this is not a bug. Please add it in new feature section

Reply:
I added the new feature section.